src/__configuration__/markdown/markdown.css

```diff
@@ -1,6 +1,22 @@
 .markdownH1:first-child {
     margin-top: 32px;
 }
+.markdownH1:first-child .MuiSvgIcon-root {
+    display: none;
```

Comment:
why no icons for first one? Click still works.

Reply:
It's usually the H1 on the very top, which feels meaningless to add a link icon. Looks like I better figure something out other than the CSS hack? Or I should just allow the first H1 to also be clicked for consistency

Reply:
I think it's fine for now...more just curious if there was a rationale. Keeping the page title clean seems like a good enough reason for me